py/selenium/webdriver/chrome/remote_connection.py

```diff
@@ -25,3 +25,4 @@ def __init__(self, remote_server_addr, keep_alive=True):
         self._commands["launchApp"] = ('POST', '/session/$sessionId/chromium/launch_app')
         self._commands["setNetworkConditions"] = ('POST', '/session/$sessionId/chromium/network_conditions')
         self._commands["getNetworkConditions"] = ('GET', '/session/$sessionId/chromium/network_conditions')
+        self._commands['ExecuteCDP'] = ('POST', '/session/$sessionId/goog/cdp/execute')
```

Comment:
Should this be camelCase to match the above commands?

I am not the expert here so maybe Lucas or David can chime in.